### PR TITLE
Add files to support 1ES setup

### PR DESCRIPTION
**Pull request by captainsafia**
Allows us to run component governance and other compliance checks on this repo.

### CODE REVIEW

File: ci.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  # Repo: 1ESPipelineTemplates/1ESPipelineTemplates
+  - repository: 1esPipelines
+    type: git
+    name: 1ESPipelineTemplates/1ESPipelineTemplates
+    ref: refs/tags/release
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1esPipelines
+  parameters:
+    sdl:
+        policheck:
```
Comment (wtgodbe) [inline ci.yml:23]: Indented 1 too many times

File: ci.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Configure which branches trigger builds
+trigger:
+  batch: true
+  branches:
+    include:
+    - main
+
+variables:
+- name: TeamName
+  value: AspNetCore
```
[the remark this answers is not in the document]
Comment (captainsafia) [reply ci.yml]: Done!

File: ci.yml (hunk 1 above)
Comment (captainsafia) [reply ci.yml]: I'll run a formatter through this file and commit the changes.

File: ci.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    - stage: buildStage
+      displayName: Build Stage
+      jobs:
+      - job: Build
+        displayName: Windows Build
+        timeoutInMinutes: 90
+        workspace:
+          clean: all
+        steps:
+        - script: dotnet build eShop.Web.slnf
```
Comment (wtgodbe) [inline ci.yml]: Need to install dotnet first: https://github.com/dotnet/arcade/blob/31624193093a13f765ab5382509e693911264509/azure-pipelines-codeql.yml#L51-L54

**Review by wtgodbe — APPROVED** [review]

**Pull request merged**